docs/user_guide.md

```diff
@@ -155,7 +155,7 @@ There may be a bit of delay during replication based on the situation of the net
 **Note:** Due to API changes, replication between different versions of Harbor may be broken.
 
 ### Creating a replication rule
-Login as a system administrator user, click `NEW REPLICATION RULE` under `Administration->Replications` and fill in the necessary fields. You can choose different replication modes, [resource filters](#resource-filter) and [trigger modes](#trigger-mode) according to the different requirements. If there is no endpoint available in the list, you need to create one. Click `SAVE` to create a replication rule.  
+Login as a system administrator user, click `NEW REPLICATION RULE` under `Administration->Replications` and fill in the necessary fields. You can choose different replication modes, [resource filters](#resource-filter) and [trigger modes](#trigger-mode) according to the different requirements. If there is no endpoint available in the list, follow the instructions in the [Installation Guide](installation_guide.md) to create one. Click `SAVE` to create a replication rule.  

```

Comment:
The `create a new endpoint here` means to add a registry endpoint in Harbor, not to setup a Harbor instance
![new_endpoint](https://user-images.githubusercontent.com/5835782/64003122-2fb03a00-cb3e-11e9-991c-5afe48cb2820.png)
After clicking the `Endpoints` link, the page is going to:
![new_endpint_2](https://user-images.githubusercontent.com/5835782/64003243-6be39a80-cb3e-11e9-95e1-20c78b20171d.png)